src/utils/project-helpers.ts

```
@@ -0,0 +1,803 @@
+  formatDate: (date: string | Date, format: 'short' | 'long' | 'relative' = 'short'): string => {
+    const dateObj = typeof date === 'string' ? new Date(date) : date;
+    
+    switch (format) {
+      case 'long':
+        return dateObj.toLocaleDateString('en-US', {
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric'
+        });
+      case 'relative':
+        const now = new Date();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)